scopes/preview/preview/preview.main.runtime.tsx

```diff
@@ -659,7 +664,14 @@ export class PreviewMain {
     const [name, uiRoot] = this.getUi();
     const resolvedAspects = await this.resolveAspects(PreviewRuntime.name, undefined, uiRoot);
     const filteredAspects = this.filterAspectsByExecutionContext(resolvedAspects, context, aspectsIdsToNotFilterOut);
-    const filePath = await this.ui.generateRoot(filteredAspects, name, 'preview', PreviewAspect.id);
+    const filePath = await this.ui.generateRoot(
+      filteredAspects,
+      name,
+      'preview',
+      PreviewAspect.id,
+      undefined,
+      (context as any).capsuleNetwork ? undefined : uiRoot.path // check if we are running inside a workspace or a capsule
```

Comment:
do this outside the function call (and maybe calc here the final path (see comment below)